DECISIONLOG.md

```diff
@@ -30,4 +30,27 @@ tests:
   ad_reporting:
     not_null_ad_reporting__account_report_account_id:
       +enabled: false
-```
\ No newline at end of file
+```
+
+### Timezone Considerations
+
+The table below documents timezone differences across platforms in the `dbt_ad_reporting` package. These differences exist due to the following reasons:
+
+- Ad platforms send pre-aggregated data that cannot be back-calculated to a finer granularity to account for timezone differences. 
+- Some platforms offer hourly report data, which could potentially be standardized. However, this approach does not ensure full coverage due to non-standard timezones, such as ±30-minute and ±45-minute offsets.
+
+Although this presents challenges within this package, customers can achieve standardization by configuring all Ad accounts and connectors to UTC whenever possible.
+
+| Platform | Grain of Data | Timezone | Hourly Custom Reports? | Can Configure Time Zones for Custom Reports? |
```

Comment:
Kinda wondering if this is too much information...thoughts on limiting this table to
Platform, Grain of Source Data, and Timezone?

I worry that talking about custom reports, given that they're not used in the package, may add some confusion here